.claude/hooks/track.sh

```
@@ -0,0 +1,26 @@
+# follow-up in the active TASKS.md (see the task-tracking skill), rather than
+# spawning a background task chip. Stateless: emits one directive per matching
+# message and nothing otherwise.
+
+set -euo pipefail
+
+input=$(cat)
+prompt=$(printf '%s' "$input" | jq -r '.prompt // empty' 2>/dev/null || printf '')
+
+# Sentinel: `$track ...` anywhere, or a line starting with `track: ...`.
+task=$(printf '%s\n' "$prompt" \
+  | grep -ioE '(\$track|^[[:space:]]*track:)[[:space:]]*.*' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)